debian/rules

```diff
@@ -10,5 +10,8 @@ override_dh_auto_configure:
 override_dh_auto_test:
 	SWTPM_TEST_SECCOMP_OPT="--seccomp action=none" make -j4 check VERBOSE=1
 
+override_dh_clean:
+	dh_clean --exclude=man/man8/swtpm-localca.8
+
```

Comment:
This is unfortunately the least nasty way to keep this static file from being deleted by `dh(1)`'s `clean` target as a result of the `man/man8/*.8` entry in `debian/clean`. `debian/clean` doesn't support exclusion entries as `.gitignore` does.